### DISCUSSION TITLE
Using lftp there is an unlegible command getting a 502 Command not allowed

### DISCUSSION THREAD

**Discussion by andreswatson** [QuorumDMS/ftp-srv#251] [Q&A]
I'm trying to connecto to my ftp-srv using lftp and once the connection is established, I see at the logs that there is a command or directive unlegible that gets a 502 error message, is it possible to understand what can be?

Here the ftp-srv logs:

{"name":"ftpLogger","hostname":"nodejs-s-1vcpu-1gb-nyc3-01","pid":1599968,"id":"0a9bc481-1cbe-497d-8918-97776f06db5f","ip":"201.216.223.162","level":10,"port":2121,"encoding":"utf8","message":"230 User logged in, proceed","msg":"Reply","time":"2021-04-02T22:42:44.921Z","v":0}
{"name":"ftpLogger","hostname":"nodejs-s-1vcpu-1gb-nyc3-01","pid":1599968,"id":"0a9bc481-1cbe-497d-8918-97776f06db5f","ip":"201.216.223.162","level":10,"msg":"[ 'PASV' ]","time":"2021-04-02T22:42:45.078Z","v":0}
{"name":"ftpLogger","hostname":"nodejs-s-1vcpu-1gb-nyc3-01","pid":1599968,"id":"0a9bc481-1cbe-497d-8918-97776f06db5f","ip":"201.216.223.162","directive":"PASV","level":10,"command":{"directive":"PASV","arg":null,"flags":[],"raw":"PASV"},"msg":"Handle command","time":"2021-04-02T22:42:45.080Z","v":0}
{"name":"ftpLogger","hostname":"nodejs-s-1vcpu-1gb-nyc3-01","pid":1599968,"id":"0a9bc481-1cbe-497d-8918-97776f06db5f","ip":"201.216.223.162","level":20,"port":65503,"msg":"Passive connection listening","time":"2021-04-02T22:42:45.098Z","v":0}
{"name":"ftpLogger","hostname":"nodejs-s-1vcpu-1gb-nyc3-01","pid":1599968,"id":"0a9bc481-1cbe-497d-8918-97776f06db5f","ip":"201.216.223.162","level":10,"port":2121,"encoding":"utf8","message":"227 PASV OK (scanntech,stage,guayerd,com,255,223)","msg":"Reply","time":"2021-04-02T22:42:45.098Z","v":0}
{"name":"ftpLogger","hostname":"nodejs-s-1vcpu-1gb-nyc3-01","pid":1599968,"id":"0a9bc481-1cbe-497d-8918-97776f06db5f","ip":"201.216.223.162","level":10,"msg":"[ '���' ]","time":"2021-04-02T22:42:45.251Z","v":0}
{"name":"ftpLogger","hostname":"nodejs-s-1vcpu-1gb-nyc3-01","pid":1599968,"id":"0a9bc481-1cbe-497d-8918-97776f06db5f","ip":"201.216.223.162","directive":"���","level":10,"command":{"directive":"���","arg":null,"flags":[],"raw":"���"},"msg":"Handle command","time":"2021-04-02T22:42:45.251Z","v":0}
{"name":"ftpLogger","hostname":"nodejs-s-1vcpu-1gb-nyc3-01","pid":1599968,"id":"0a9bc481-1cbe-497d-8918-97776f06db5f","ip":"201.216.223.162","level":10,"port":2121,"encoding":"utf8","message":"502 Command not allowed: ���","msg":"Reply","time":"2021-04-02T22:42:45.251Z","v":0}
{"name":"ftpLogger","hostname":"nodejs-s-1vcpu-1gb-nyc3-01","pid":1599968,"id":"0a9bc481-1cbe-497d-8918-97776f06db5f","ip":"201.216.223.162","level":10,"msg":"Passive server closed","time":"2021-04-02T22:42:45.252Z","v":0}

**Comment by matt-forster** [comment]
From your logs, it looks like the command is being passed into the server as non-utf characters? 
The diamonds in your log show the command - and it is not recognized, so the server does not know what to do.
